scripts/docs/build-html-docs.ts

```diff
@@ -0,0 +1,28 @@
+import { Application, TSConfigReader, TypeDocOptions } from "typedoc";
+
+interface HtmlDocsOptions {
+    options?: Partial<TypeDocOptions>;
+    outputDir: string;
+}
+
+export const buildHtmlDocs = async ({ outputDir, options }: HtmlDocsOptions) => {
+    const app = new Application();
+
+    // So TypeDoc can load tsconfig.json
+    app.options.addReader(new TSConfigReader());
+
+    app.bootstrap({
+        ...options,
+        theme: 'minimal',
+        readme: 'none',
```

Comment:
nit:  did you consider adding docs/index.md as the index file?  I tried it and it seems to work, but, we'd need to mutate the relative ./ClientApp.md links here as well as we do in the dev-center.

Just curious on your thoughts here.  It's a nit and we def don't need to add it as we don't have a consumer of the html docs at this point.